### PR TITLE
#179 Support tracing in katharsis

### PR DESCRIPTION
**Pull request by remmeier**
added integration into zipkin/brave with a new "BraveModule" hosted in katharsis-brave. The module can simply be added to KatharsisClient and will integrate Zipkin/Brave accordingly.

### CODE REVIEW

**Comment by remmeier** [comment]
something strange going on with the builds, the pull request would like to create a release:

What is the release version for "Katharsis parent POM"? (io.katharsis:katharsis-parent) 2.8.2: :

File: katharsis-brave/pom.xml [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
```
Comment (sonarqube-reporter) [inline katharsis-brave/pom.xml]: MINOR Tab characters found (this is the first occurrence) [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)

File: katharsis-brave/pom.xml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		
+		<dependency>
+			<groupId>io.zipkin.brave</groupId>
+			<artifactId>brave-okhttp</artifactId>
+			<version>${brave.version}</version>
+			<scope>compile</scope>
+		</dependency>
+		
+		<dependency>
+            <groupId>com.google.guava</groupId>
```
Comment (sonarqube-reporter) [inline katharsis-brave/pom.xml]: MINOR Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)

File: katharsis-brave/pom.xml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+        </dependency>
+        
+		<!-- Test Dependencies -->
+	    <dependency>
```
Comment (sonarqube-reporter) [inline katharsis-brave/pom.xml]: MINOR Make this line start at column 5. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)

**Comment by sonarqube-reporter** [comment]
SonarQube analysis reported 6 issues
* MAJOR 1 major
* MINOR 5 minor

Watch the comments in this conversation to review them.

**Pull request merged**